src/applications/appeals/995/tests/995-supplemental-claim.cypress.spec.js

```diff
@@ -58,6 +58,32 @@ const testConfig = createTestConfig(
           .should('not.exist');
         cy.get('select').select('Buddy/Lay Statement');
       },
+      'contestable-issues': ({ afterHook }) => {
+        cy.fillPage();
+        cy.injectAxeThenAxeCheck();
+        afterHook(() => {
+          cy.get('@testData').then(testData => {
+            testData.additionalIssues.forEach(({ issue, decisionDate }) => {
+              if (issue) {
+                cy.get('.add-new-issue').click();
+                cy.url().should('include', `${BASE_URL}/add-issue?index=`);
+                cy.axeCheck();
+                cy.get('#add-sc-issue')
+                  .shadow()
+                  .find('input')
+                  .type(issue);
+                const date = decisionDate
+                  .split('-')
+                  .map(v => parseInt(v, 10))
+                  .join('-');
```

Comment:
We did this because the `cy.fillDate` command was failing to find the option. I feel like the `va-select` component was recently updated to remove the leading zeros? The `cy.fillDate` command will need to be updated.

Reply:
Nope, I was wrong. The `va-select` has not ever included leading-zeros. So the `cy.fillDate` will need to be updated.